src/content/en/updates/posts/2016/05/houdini.md

```diff
@@ -0,0 +1,366 @@
+---
+layout: updates/post
+title: "Houdini – Demystifying CSS"
+description: "Houdini is a collection of APIs that expose the CSS engine’s internals to developers"
+published_on: 2016-05-20
+updated_on: 2016-05-20
+authors:
+  - surma
+tags:
+  - houdini
+  - css
+featured_image: /web/updates/images/2016/05/houdini/compworklet_small.png
+
+---
+
+Have you ever thought about the amount of work CSS does? You change a single
+attribute and suddenly your entire website appears in a different layout. It’s
+kind of *magic* in that regard (can you tell where I am going with this?!). So
+far, we – the community of web developers – have only been able to witness and
+observe the magic. What if we want to come up with our own magic tricks and
+perform them? What if we want to *become the magician*? Enter Houdini!
+
+The Houdini task force consists of engineers from Mozilla, Apple, Opera,
+Microsoft, HP, Intel and Google working together to expose certain parts of the
+CSS engine to web developers. The task force is working on a *collection of
+drafts* with the goal to get them accepted by the W3C to become actual web
```

Comment:
Does "web standard" always imply W3C acceptance? This phrasing makes it sound that way.

(I don't know whether there's actually anything wrong with your phrasing, but it might be something to run by someone who's been involved in the standards process.)